website/content/docs/commands/token/lookup.mdx

```diff
@@ -8,9 +8,47 @@ description: |-
 
 # token lookup
 
+Displays information about a token or accessor.
+
+<CodeBlockConfig hideClipboard>
+
+```shell-session
+$ vault token lookup [flags] <token>
+
+$ vault token lookup [-help | -h]
+```
+
+</CodeBlockConfig>
+
+## Description
+
 The `token lookup` displays information about a token or accessor. If a TOKEN is
 not provided, the locally authenticated token is used.
 
+## Command arguments
+
+<br />
+
+@include 'cli/token/args/token.mdx'
+
+## Command options
+
+- None
+
+## Command flags
+
+<br />
+
+@include 'cli/token/flags/accessor.mdx'
+
+<br /><hr /><br />
+
+@include 'cli/token/flags/format.mdx'
+
+## Standard flags
+
+@include 'cli/standard-settings/all-standard-flags-but-format.mdx'
```

Comment:
`vault token lookup` supports the `-format` flag, so the include should be `@include 'cli/standard-settings/all-standard-flags.mdx'`